src/OpenTelemetry.Api/README.md

```diff
@@ -170,8 +170,9 @@ required only for the following scenarios:
 
 As mentioned in the introduction, the instrumentation API for OpenTelemetry .NET
 is the .NET `Activity` API. Guidance for instrumenting using this API is
-documented fully in the [activity user guide](https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.DiagnosticSource/src/ActivityUserGuide.md), but is described
-here as well.
+documented fully in the 
+[activity user guide](https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.DiagnosticSource/src/ActivityUserGuide.md), 
```

Comment:
this is the old link, which doesn't contain the current recommendation of using `ActivitySource`.

https://docs.microsoft.com/dotnet/core/diagnostics/distributed-tracing#getting-started-with-tracing is the right link.